cloud/cluster/state.go

```diff
@@ -37,13 +39,15 @@ func (s *state) setAndDiff(newState []Node) []NodeUpdate {
 	sort.Sort(NodeSorter(removed))
 	outgoing := []NodeUpdate{}
 	for _, n := range added {
+		log.Infof("NodeAdded update: %s", n)
```

Comment:
should the formatting directive be %v?

Reply:
nope - n is a Node, and Node has a String() method, so %s should work fine